backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/contract/moduleLog/contractModuleLogSalesOrder.feature

```diff
@@ -42,9 +42,10 @@ Feature: Modular contract log from sales order
       | year_2023            | 2023       | harvesting_calendar      |
 
     And metasfresh contains ModCntr_Types:
-      | ModCntr_Type_ID.Identifier | Name            | Value           | Classname                                                               |
-      | modCntr_type_PO            | modCntr_type_PO | modCntr_type_PO | de.metas.contracts.modular.impl.PurchaseOrderLineModularContractHandler |
-      | modCntr_type_SO            | modCntr_type_SO | modCntr_type_SO | de.metas.contracts.modular.impl.SalesOrderLineModularContractHandler    |
+      | ModCntr_Type_ID.Identifier | Name              | Value             | Classname                                                               |
+      | modCntr_type_PO            | modCntr_type_PO   | modCntr_type_PO   | de.metas.contracts.modular.impl.PurchaseOrderLineModularContractHandler |
+      | modCntr_type_SO            | modCntr_type_SO   | modCntr_type_SO   | de.metas.contracts.modular.impl.SalesOrderLineModularContractHandler    |
+      | modCntr_type_SHIP          | modCntr_type_SHIP | modCntr_type_SHIP | de.metas.contracts.modular.impl.ShipmentLineModularContractHandler      |
```

Comment:
why add `modCntr_module_SHIP`  to this scenario?

Reply:
this is what causes the error, because we can not change contract setting when it's already used in ongoing contract